e2e/fixtures-firefox.ts

```
@@ -0,0 +1,35 @@
+import type { BrowserContext } from '@playwright/test'
+import path from 'node:path'
+import { test as base, firefox } from '@playwright/test'
+
+export const test = base.extend<{
+  context: BrowserContext
+  extensionId: string
+}>({
+  // eslint-disable-next-line no-empty-pattern
+  context: async ({}, _useContext) => {
```

Biome (1.9.4)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)